examples/lessons/11/app.js

```diff
@@ -77,7 +77,8 @@ export default class AppAnimationLoop extends AnimationLoop {
   }
 
   onInitialize({canvas, gl}) {
-    addMouseHandler(canvas);
+    const eventManager = new EventManager(canvas);
```

Comment:
One line comment explaining why we went with mjolnir (as opposed to built in event handling?)